internal/cmap/cmap.go

```diff
@@ -116,8 +119,13 @@ type CMap struct {
 	cached []byte
 }
 
-// NewToUnicodeCMap returns an identity CMap with codeToUnicode matching the `codeToUnicode` arg.
-func NewToUnicodeCMap(codeToUnicode map[CharCode]rune) *CMap {
+// NewToUnicodeCMap returns an identity CMap with codeToUnicode matching the `codeToRune` arg.
+func NewToUnicodeCMap(codeToRune map[CharCode]rune) *CMap {
```

Comment:
Should we consider changing the parameter to `codeToString map[CharCode]string`?

Reply:
That would break a lot of code as this function gets called from 6 places.  

I thought it best to localise changes for now. It should be straightforward to do a second PR for this specific change, if you want this change.

Reply:
codeToRune seems better in most cases its the code <-> rune that matters, the multi rune cases (string) are kind of an exception

Reply:
The codeToRune seems more clear to me.  Can be reconsidered in the future if we have more "exceptions".